package.json

```diff
@@ -90,7 +90,7 @@
     "politespace": "^0.1.4",
     "run-sequence": "^1.1.5",
     "simplecrawler": "^1.1.3",
-    "uswds": "1.3.0",
+    "uswds": "18F/web-design-standards#fb49e4f",
```

Comment:
Ok, so I'm tempted to say that this is how we should specify the USWDS version--either by a specific commit hash, or by a version tag... in this case I think we need the commit hash because our latest 1.3.0 release might not have all the fractal components we need?

Anyways, i hesitate to put just `18F/web-design-standards` because then we have no idea what version of the standards it's actually using...